CMakeLists.txt

```diff
@@ -6,11 +6,13 @@ catkin_simple()
 
 include(ExternalProject)
 
+set(YAML_CPP_TAG 11607eb5bf1258641d80f7051e7cf09e317b4746)
```

Comment:
Ok, sounds good. But could you please add a comment for L9 describing the reasons why we fix to a non-release commit?